docs/assets/stylesheets/_styles.scss

```diff
@@ -0,0 +1,29 @@
+// =============================================================================
+// Styles
+// =============================================================================
+
+main {
+  font-family: "Quicksand";
+  letter-spacing: -.06em;
+}
+
+header {
+  display: block;
+  padding: 150px 0;
+  background: linear-gradient(to right, #e53935 0%, #ec7574 100%);
```

Comment:
Color literals like `#e53935` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#ec7574` should only be used in variable declarations; they should be referred to via variable everywhere else.